packages/electrode-archetype-react-app-dev/config/webpack/add-dll-references.js

```diff
@@ -25,7 +36,17 @@ module.exports = function (config) {
       }));
     }
   } catch (err) {
-    console.log("add-dll-references failed", err);
+    winston.add(winston.transports.File, {
+        filename: 'exceptions.log',
+        handleExceptions: true,
+        humanReadableUnhandledException: true
+      }
+    );
```

Comment:
Is adding a new transport at error-time good practice? It seems like it would make more sense to add a special transport during configuration that logs a certain level to `exceptions.log`.